bash/fzf-bash-completion.sh

```diff
@@ -120,7 +122,7 @@ EOF
 }
 
 _fzf_bash_completion_compspec() {
-    complete -p -- "$1" || complete -p '' || echo complete -o filenames -F _fzf_bash_completion_fallback_completer
+    complete -p -- "$1" || complete -p '' || printf '%s\n' complete -o filenames -F _fzf_bash_completion_fallback_completer
```

Comment:
this going to print each of the `complete`, `-o`, `filenames` ... on a separate line, it should be space separated

Reply:
yeah you're right, good shout. have fixed by wrapping the `complete -o...` in quotation marks